tests/base/shared/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright 2022-2024 MosaicML Streaming authors
```

Comment:
Mind deleting this file and moving the tests in `tests/base/shared/test_array.py` to the existing `tests/test_shared.py`? Ty!

Reply:
Sounds good! Fixed!